### DISCUSSION TITLE
NEW SKIOVOX VERSION 124

### DISCUSSION THREAD

**Discussion by Xiaoethan** [3kh0/ext-remover#1098] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
it should work for all districts since its not like a heavy exploit, DM zaerq on discord for method / i have proof

**Comment by Xiaoethan** [comment]
Me and my friends have found this so, we are replaceing the name with skibidivox.

**Reply by S-PScripts** [reply]
> Me and my friends have found this so, we are replaceing the name with skibidivox.

don't you shucking dare name it that

edit: is this https://github.com/Brandon421-ops/Exploits-And-Hacks?tab=readme-ov-file#requirements-5

**Reply by idkanymorewtf** [reply]
it's gone bud

**Reply by S-PScripts** [reply]
> it's gone bud

ok? why did you need to necropost?
